servers/src/main/java/tachyon/worker/block/UserCleanup.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tachyon.Constants;
+import tachyon.conf.TachyonConf;
+import tachyon.util.CommonUtils;
+
+/**
+ * UserCleanup periodically checks if any user have become zombies,
+ * removes the zombie user and associated data when necessary
+ * The syncing parameters (intervals) adopt directly from worker-to-master heartbeat
+ * configurations.
+ */
+public class UserCleanup implements Runnable {
```

Comment:
NIT: How about naming this class to something like `zombieUserCleaner` or maybe just `userCleaner`, to be more consistent with current naming convention?